OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -786,10 +787,33 @@ public void ResolveOrder(Actor self, Order order)
 				else
 					self.QueueActivity(new HeliReturnToBase(self, Info.AbortOnResupply, null, false));
 			}
+			else if (order.OrderString == "Scatter")
+				MoveToRandomCell(self);
 		}
 
 		#endregion
 
+		void MoveToRandomCell(Actor self)
```

Comment:
I interpret this name as "a random cell anywhere on the map", which of course isn't what this does.

Can you please rename this to either `Nudge` (for consistency with `Mobile`) or perhaps more appropriately `Scatter`?

Reply:
That is true... Changed this to Nudge.